content/docs/getting-started/create-proxy/examples/fragments/index.md

```diff
@@ -0,0 +1,116 @@
+---
```

Comment:
I'm not getting started is the place for this kind of content. Maybe in-depth is a better fit?